infrastructure/kzg/src/main/java/tech/pegasys/teku/kzg/RustKZG.java

```diff
@@ -102,6 +103,13 @@ public KZGProof computeBlobKzgProof(final Bytes blob, final KZGCommitment kzgCom
     throw new RuntimeException("LibPeerDASKZG library doesn't support computeBlobKzgProof");
   }
 
+  // TODO: test
```

Comment:
Do you plan to test it as part of this PR? Otherwise it would be good to add a ticket link here so we know we won't miss it.